### PR TITLE
Use issue number as value of issue sections

### PR DESCRIPTION
**Pull request by tarsius**
Needed for e.g. https://github.com/tarsius/magit-rockstar/commit/d7458d919cec301c8c293b6a30b1beaea06f718d. You could of course also store the complete plist.

### CODE REVIEW

File: magithub-issue.el [hunk 1]
```diff
@@ -157,7 +157,7 @@ Returns a plist with the following properties:
 (defun magithub-issue--insert (issue)
   "Insert an `issue' as a Magit section into the buffer."
   (when issue
-    (magit-insert-section (magithub-issue issue)
+    (magit-insert-section (magithub-issue issue (plist-get issue :number))
```
Comment (vermiculus) [inline magithub-issue.el]: Isn't that the `HIDE` position?

```
(magit-insert-section [NAME] (TYPE &optional VALUE HIDE) &rest BODY)
```

Comment (vermiculus) [inline magithub-issue.el]: Besides, https://github.com/tarsius/magit-rockstar/commit/d7458d919cec301c8c293b6a30b1beaea06f718d seems like it's using the complete plist (i.e., `issue`) anyway.

**Comment by tarsius** [comment]
It was late...

**Comment by vermiculus** [comment]
I know the feeling 👍

**Pull request closed without merging**